src/System.Runtime.Extensions/src/System/AppDomainUnloadedException.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*=============================================================================
+**
+** 
+**
+** Purpose: Exception class for attempt to access an unloaded AppDomain
+**
+**
+=============================================================================*/
+
+namespace System {
+
+    using System.Runtime.Serialization;
+
+    [System.Runtime.InteropServices.ComVisible(true)]
```

Comment:
It is useless in .NET Core. We just didn't get to nuking it from everywhere in corefx.